### PR TITLE
Update free-courses-en.md

### PR DESCRIPTION
**Pull request by apu52**
## What does this PR do?
Add resource(s) 

### Description
 I have added a very well known & efficient playlist for graph theory.
### Why is this valuable (or not)?
It is valuable
### How do we know it's really free?
It is free of cost
### For book lists, is it a book? For course lists, is it a course? etc.
It is a course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by apu52** [comment]
@eshellman Please take a look at my PR & please merge it.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -733,6 +733,7 @@
 * [Graph Theory](https://youtube.com/playlist?list=PLztBpqftvzxXBhbYxoaZJmnZF6AUQr1mH) - Wrath of Math
 * [Graph Theory - DM](https://youtube.com/playlist?list=PLYrahs7hsYIQiSNxTfZndQz7jWPXsA1ur) - SCCI Labs IIT Ropar
 * [Graph Theory Playlist](https://youtube.com/playlist?list=PLDV1Zeh2NRsDGO4--qE8yH72HFL1Km93P) - WilliamFiset
+* [Graph Theory Playlist](https://www.youtube.com/playlist?list=PLC36xJgs4dxHYyeppopMuALQhKY0ADIHi) - GATE CSE LECTURES BY AMIT KHURANA
```
Comment (sohansai) [inline courses/free-courses-en.md:736]: Use the same title from the provided playlist link and keep the link in a single line.

**Comment by apu52** [comment]
@sohansai I have updated the name but I can't be able to make it in a line as it is not possible to be fit in one line...Can you please check my PR & please merge it...

**Comment by sohansai** [comment]
> @sohansai I have updated the name but I can't be able to make it in a line as it is not possible to be fit in one line...Can you please check my PR & please merge it...

It's already in one line; I haven't seen it. It looks good now

**Comment by sohansai** [comment]
Use the same title from the provided playlist link, include the creator's name, and review docs/CONTRIBUTING.md before contributing

**Comment by eshellman** [comment]
changes already merged

**Pull request closed without merging**